plugins/devtools/README.md

```diff
@@ -418,8 +418,8 @@ By default, only packages with names starting with `@backstage` and `@internal`
 devTools:
   info:
     packagePrefixes:
-      - @roadiehq/backstage-
-      - @spotify/backstage-
+      - '@roadiehq/backstage-'
+      - '@spotify/backstage-'
```

Comment:
Wonder if it's worth actually calling out that you need the quotes?

Reply:
Yes, I think it's necessary because if you don't put quotes, it doesn't work, there is a YAMLParseError when you start Backstage with `yarn dev`

Reply:
Sorry, I mean adding a comment saying the quotes a required? If someone doesn't copy this but types it in and leaves off the quotes they hit the error.